### PR TITLE
Feature/create tables crud

### PR DESCRIPTION
**Pull request by Kosik33i6**

### CODE REVIEW

File: models/Table.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import mongoose from "mongoose";
+const { Schema } = mongoose;
+
+const tableSchema = new mongoose.Schema({
+  id: Schema.Types.ObjectId,
+  name: {
```
Comment (radekwojpl2) [inline models/Table.js:6]: add restaurant id here

File: routes/tableRouter.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import { tableController } from '../controllers/tableController.js';
+
+export const tableRouter = express.Router();
+
+tableRouter
+  .get('/', tableController.getTables)
+  .post('/', tableController.addTable)
+  .get('/:id', tableController.getTable)
+  .put('/:id', tableController.updateTable)
+  .delete('/:id', tableController.deleteTable);
```
Comment (radekwojpl2) [inline routes/tableRouter.js:11]: add endpoint for getting all tables within one restaurant

File: models/Table.js (hunk 1 above)
Comment (Kosik33i6) [reply models/Table.js:6]: Instead of the id that is now, should I add the id of the restaurant?

File: routes/tableRouter.js (hunk 2 above)
Comment (Kosik33i6) [reply routes/tableRouter.js:11]: I don't know how to do this and I need help

**Pull request merged**